src/network/networkd-link.c

```diff
@@ -1451,7 +1465,7 @@ static int link_acquire_ipv6_conf(Link *link) {
                         return log_link_warning_errno(link, r,  "Could not acquire DHCPv6 lease: %m");
         }
 
-        if (link_ipv6_accept_ra_enabled(link)) {
+        if (link_ipv6_accept_ra_enabled(link) && !link_is_bridge_port(link)) {
```

Comment:
This is probably unnecessary, since when no link-local address is configured, networkd will not do SLAAC.